spec/requests/api/v5/sections_spec.rb

```diff
@@ -56,4 +56,51 @@ def json_validate_sections(sections=@sections, periods=false)
       json_validate_sections(Section.all, true)
     end
   end
+  context 'There is a section to be updated' do
+    it 'updates the seats_taken for section' do
+      section = FactoryGirl.create(:section_with_periods, seats_taken: 0)
+      #put "/api/v5/sections/#{section.id}", seats_taken: 1 
+      #binding.pry
+      section_params = {
+        section: {
+          seats_taken: 2
+         # name:"4"
+         # crn:"CSCI-1200"
+         # seats: 300
+         # instructors:"William Thompson"
+         # num_periods: 2
+         # course_id: 332
+        }
+      }
+      put "/api/v5/sections/#{section.id}", section_params
+      section.reload
+      expect(section.seats_taken).to eq 2
+    end
+    it 'updates the seats for section' do
+      section_params = { 
+        section: {
+          seats: 40
+        }
+      }
+
+      section = FactoryGirl.create(:section_with_periods, seats: 20)
+      put "/api/v5/sections/#{section.id}", section_params
+      section.reload
+      expect(section.seats).to eq 40    
+    end
+
+    it 'deletes a section' do
+      section = FactoryGirl.create(:section_with_periods)
+      #put "/api/v5/sections/#{section.id}", section_params
+      delete "/api/v5/sections/#{section.id}"
+      expect(response.status).to eq 204
+    end
+
+    it 'session id does not exist' do
```

Comment:
please make this test name more descriptive. Consider grouping the put tests in one context and the delete tests in another